stdlib/public/core/CTypes.swift

```diff
@@ -43,14 +43,14 @@ public typealias CShort = Int16
 public typealias CInt = Int32
 
 /// The C 'long' type.
-#if os(Windows) && !CYGWIN && arch(x86_64)
+#if os(Windows) && !os(Cygwin) && arch(x86_64)
```

Comment:
Old `os(Windows) && !CYGWIN` is equivalent to new `os(Windows)`.
Just drop `&& !CYGWIN`.
Change to `#if os(Windows) && arch(x86_64)`